### PR TITLE
Fy data can be write into Endf6 format

### PR DESCRIPTION
**Pull request by AitorBengoechea**

### CODE REVIEW

**Review by luca-fiorito-11 — CHANGES_REQUESTED** [review]

File: sandy/fy.py [hunk 1]
```diff
@@ -279,6 +279,48 @@ def from_endf6(cls, endf6, verbose=False):
...
+        >>> tape = sandy.get_endf6_file("jeff_33", "nfpy", 922350)
+        >>> fy = sandy.Fy.from_endf6(tape)
+        >>> new_tape = fy.to_endf6(tape)
+        >>> assert new_tape.data == tape.data
+        """
+        endf6new = self._NPFY_to_endf6(endf6, 454)
+        endf6new = self._NPFY_to_endf6(endf6new, 459)
+        return endf6new
+
+    def _NPFY_to_endf6(self, endf6, nfpy_data):
```
[the remark this answers is not in the document]
Comment (AitorBengoechea) [reply sandy/fy.py:311]: I change the function

File: sandy/fy.py [hunk 2]
```diff
@@ -279,6 +279,48 @@ def from_endf6(cls, endf6, verbose=False):
         df = pd.DataFrame(data)
         return cls(df)
 
+    def to_endf6(self, endf6):
+        """
+        Update cross sections in `Endf6` instance with those available in a
+        `Xs` instance.
+
+        .. warning:: only nfpy IFY and CFY that are originally
+                     present in the `Endf6` instance are modififed
```
[the remark this answers is not in the document]
Comment (AitorBengoechea) [reply sandy/fy.py:288]: I think that i solve

**Comment by AitorBengoechea** [comment]
@luca-fiorito-11  i upload the new version. I dont close one change because i dont understand what you are meaning

**Comment by AitorBengoechea** [comment]
@luca-fiorito-11 done last changes

**Review by luca-fiorito-11 — APPROVED** [review]

**Pull request merged**